src/components/ha-form/ha-form-string.ts

```diff
@@ -138,15 +138,10 @@ export class HaFormString extends LitElement implements HaFormElement {
       }
       ha-icon-button {
         position: absolute;
-        top: 1em;
-        right: 12px;
-        --mdc-icon-button-size: 24px;
+        top: 8px;
+        inset-inline-end: 8px;
```

Comment:
It is fine to add this, but we can't remove the `right` property, as `inset-inline` is not supported by all of our supported browsers.